trading/trader/modes/abstract_trading_mode.py

```diff
@@ -51,17 +51,38 @@ def __init__(self, config, exchange):
     def get_name(cls):
         return cls.__name__
 
+    @classmethod
+    def get_config_folder(cls):
+        return f"{TENTACLES_PATH}/{TENTACLES_TRADING_PATH}/{TENTACLES_TRADING_MODE_PATH}/{EVALUATOR_CONFIG_FOLDER}"
+
     @classmethod
     def get_config_file_name(cls):
-        return f"{TENTACLES_PATH}/{TENTACLES_TRADING_PATH}/{TENTACLES_TRADING_MODE_PATH}/{EVALUATOR_CONFIG_FOLDER}/" \
-               f"{cls.get_name() + CONFIG_FILE_EXT}"
+        return f"{cls.get_config_folder()}/{cls.get_name() + CONFIG_FILE_EXT}"
+
+    @classmethod
+    def get_config_file_schema_name(cls):
+        return f"{cls.get_config_folder()}/{cls.get_name()}_{SCHEMA}{CONFIG_FILE_EXT}"
+
+    @classmethod
+    def get_specific_config(cls, raise_exception=True, raw_file=False):
```

Comment:
Maybe these two function can be inherited from a new "Tentacle" abstract class ?

Reply:
Yep, that's also what I thought, I think it's better to leave it to another PR though

Reply:
Yes !